pkg/kubecfg/validate.go

```
@@ -42,7 +43,12 @@ func (c ValidateCmd) Run(apiObjects []*unstructured.Unstructured, out io.Writer)
 
 		schema, err := utils.NewSwaggerSchemaFor(c.Discovery, obj.GroupVersionKind().GroupVersion())
 		if err != nil {
-			allErrs = append(allErrs, fmt.Errorf("Unable to fetch schema: %v", err))
+			e := fmt.Errorf("Unable to fetch schema: %v", err)
+			if c.IgnoreUnknown {
```

Can we make this more specific to just "not found"?  Eg: I don't want to silently skip _all_ validation if the client can't authenticate to the apiserver or some other unrelated (global) condition.

---

done, but instead of checking the error, I check whether the GroupVersion is known (incidentally, this is the same check that is performed later on by `util.NewSwaggerSchemaFor`)